### ISSUE TITLE
polysemy-0.3.0.0 benchmark build failure

### ISSUE THREAD

**Issue by DanBurton** [isovector/polysemy#92]
As seen on the Stackage build server:

```
Building benchmark 'polysemy-bench' for polysemy-0.3.0.0..
[1 of 3] Compiling Paths_polysemy   ( dist/build/polysemy-bench/autogen/Paths_polysemy.hs, dist/build/polysemy-bench/polysemy-bench-tmp/Paths_polysemy.o )
[2 of 3] Compiling Poly             ( bench/Poly.hs, dist/build/polysemy-bench/polysemy-bench-tmp/Poly.o )

bench/Poly.hs:20:53: error:
    Not in scope: type constructor or class ‘Semantic’
   |
20 | slowBeforeSpecialization :: Member (State Int) r => Semantic r Int
   |                                                     ^^^^^^^^

bench/Poly.hs:29:44: error:
    Not in scope: type constructor or class ‘Semantic’
   |
29 | {-# SPECIALIZE slowBeforeSpecialization :: Semantic '[State Int] Int #-}
   |                                            ^^^^^^^^

bench/Poly.hs:37:8: error:
    Not in scope: type constructor or class ‘Semantic’
   |
37 |     :: Semantic '[ State Bool
   |        ^^^^^^^^

```

**Comment by isovector** [comment]
Thanks for the heads up. I've fixed it in HEAD.

**Comment by isovector** [comment]
@DanBurton fixed in `polysemy-0.3.0.1`
